src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
+
+    public Integer getQuantity(String fruit) {
+        return fruits.get(fruit);
+    }
+
+    public void put(String fruit, Integer quantity) {
+        fruits.put(fruit, quantity);
+    }
+
+    public Set<String> keyset() {
+        return fruits.keySet();
+    }
+    /*
+    The SupplyOperation class should not directly access and manipulate the Storage class.

```

Comment:
This comment block is a good tip but it doesn't belong in this class. Consider moving it to a more appropriate location, like a README or the class it's referring to.

Reply:
ok, done